extraction/data_models/embedded_instagram_post.py

```diff
@@ -0,0 +1,33 @@
+"""This class creates the object structure for Embedded Instagram content"""
+
+from data_models.contents import ContentType, _Content
+
+
+class EInstagramPost(_Content):
+    """This class creates Embedded Instagram object"""
+
+    def __init__(self, code):
+        super(EInstagramPost, self).\
+            __init__(ContentType.EMBEDDED_INSTAGRAM_POST)
+        self.insta_shortcode = code
+        self.media_type = ''
+
+    def __str__(self):
```

Comment:
Can you check if python prettyprint gives the string representation you want - https://docs.python.org/3/library/pprint.html 

Maintaining the object to string printers will become cumbersome, I think we should just remove them.

Reply:
Thanks for the suggestion. Updated!